tests/testdata/default_config.yaml

```
@@ -265,6 +265,10 @@ train:
   # Allow CPU offload for deepspeed optimizer.
   # Default: False
   deepspeed_cpu_offload_optimizer: false
+  # PyTorch device to use. Use 'cpu' for 'simple' and 'full' training.
+  # Use 'cuda' for NVidia CUDA / AMD ROCm GPUs. Use 'hpu' for Intel Gaudi GPUs.
```

```suggestion
  # Use 'cuda' for Nvidia CUDA / AMD ROCm GPUs. Use 'hpu' for Intel Gaudi GPUs.
```

just run make regenerate-testdata again once resolving the above